build_stats.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/sh
+
+set -e
+
+HERE_DIR="$(cd "$(dirname "$1")"; pwd)/$(basename "$1")"
```

Comment:
Hmm, it doesn't work properly on my system if I'm outside the directory but the script is in my path. Let me look into it.